src/components/Hero/Hero.jsx

```diff
@@ -0,0 +1,25 @@
+import styles from './Hero.module.css';
+import PropTypes from 'prop-types';
+
+export default function Hero({title, bodyText}) {
+    return (
+        <div className={styles.heroContainer}>
+            <div className={styles.container}>
+                <div className={styles.innerContainer}>
+                    <div className={styles.textElements}>
+                        <h2 className={styles.headlineTitle}>{title}</h2>
+                        <div className={styles.bodyText}>
+                            <p>{bodyText}</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
```

Comment:
Das sind ganz schön viele container 😀  Das kann man bestimmt auch mit weniger lösen!
Vorschlag: Schau dir mal CSS grid an. Du würdest in dem fall einfach ein grid mit 2 reihen anlegen und darin befindet sich dann die überschrift und der bodytext